### DISCUSSION TITLE
Checkbox value get

### DISCUSSION THREAD

**Discussion by mahmudmihon** [ant-design-blazor/ant-design-blazor#3505] [Q&A] [answered]
I have a table and I used checkbox component in a column for every row. Now if a checkbox is checked or unchecked how can I capture the state and the checkbox value?

**Comment by ElderJames** [comment, marked as answer]
Thanks for contacting us @mahmudmihon , you can refer the document about the Table cell editing.

https://antblazor.com/en-US/components/table#components-table-demo-edit-cell
